dotnet6/cookiecutter-aws-sam-quickstart-sns-dotnet/cookiecutter.json

```diff
@@ -0,0 +1,4 @@
+{
+    "project_name": "Name of the project",
+    "runtime": "dotnet6.0"
```

Comment:
@neville-lewis can you please add these proprieties to make it easy to select Graviton 2 (ARM64) architecture?

[https://github.com/aws/aws-sam-cli-app-templates/blob/6aaf6ac2e0ada5f87c4c6f69c5c1d8696513f2fe/dotnet6/cookiecutter-aws-sam-hello-step-functions-sample-app/cookiecutter.json#L4](https://github.com/aws/aws-sam-cli-app-templates/blob/6aaf6ac2e0ada5f87c4c6f69c5c1d8696513f2fe/dotnet6/cookiecutter-aws-sam-hello-step-functions-sample-app/cookiecutter.json#L4)

Reply:
Done.